src/items/traits.md

```diff
@@ -3,9 +3,9 @@
 A _trait_ describes an abstract interface that types can implement. This
 interface consists of [associated items], which come in three varieties:
 
-- [functions](items/associated-items.html#associated-functions-and-methods)
-- [types](items/associated-items.html#associated-types)
-- [constants](items/associated-items.html#associated-constants)
+- [functions]
```

Comment:
I feel like this is going to cause confusion as just being functions. How about `[function][associated functions]`?

Reply:
I don't want to make functional changes here.